src/ui/components/Timeline/index.tsx

```diff
@@ -97,17 +97,13 @@ export default function Timeline() {
 
         <div
           className="progress-bar-container"
+          onClick={onClick}
           onMouseEnter={() => setIsHovered(true)}
           onMouseLeave={() => setIsHovered(false)}
         >
           <div className="progress-bar-stack">
             <ProtocolTimeline />
-            <div
-              className="progress-bar"
-              ref={progressBarRef}
-              onClick={onClick}
-              onMouseMove={onMouseMove}
-            >
+            <div className="progress-bar" ref={progressBarRef} onMouseMove={onMouseMove}>
```

Comment:
Feels weird to me to move the click handler but not the mousemove handler. Why?

Reply:
oversight. mouseMove would be nice too